ui/src/app/service/utils.service.ts

```diff
@@ -0,0 +1,22 @@
+import {Injectable} from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+/**
+ * Common methods to be used cross all components
+ */
+export class UtilsService {
+
+  constructor() {
+  }
+
+  /**
+   * Convert proto timestamp to readable string
+   */
+  convertTimestamp(timestamp: any): string {
+    const date = new Date(timestamp.getSeconds() * 1000 + timestamp.getNanos() / 1000);
+    const month = date.toLocaleString('default', {month: 'long'});
+    return month + ' ' + date.getUTCDate() + ' ' + date.getUTCHours() + ':' + date.getUTCMinutes();
```

Comment:
`Date.toLocaleString()` will format the time when you put a specific locale instead of `'default'`.

Reply:
just fixed. ``toLocaleString()`` it's a very handy method but I use it in a wrong way :(